app/models/concerns/soft_deletable.rb

```diff
@@ -0,0 +1,10 @@
+module SoftDeletable
+  extend ActiveSupport::Concern
+
+  included do
+    default_scope { where(soft_deleted_at: nil) }
+
+    scope :include_deleted, -> { unscope(where: :soft_deleted_at) }
```

Comment:
wondering if "with_deleted" may be better here. i think of "include" being a keyword in active record that does eager loading/joins

Reply:
i did it